packages/lib/clipperUtils.ts

```diff
@@ -34,7 +34,7 @@ function baseUrl() {
 		output2.pop();
 		output = output2.join('/');
 	}
-	if (output[output.length - 1] === '/'){
+	if (output[output.length - 1] === '/') {
```

Comment:
Thanks for fixing this. We're actually trying to understand why the precommit hook doesn't always run (https://github.com/laurent22/joplin/issues/11972). Would you mind letting us know if you had any issue when running `yarn install`, and also what OS you're using?

Reply:
Ok that makes sense, thank you. I guess the pre-commit hook cannot always be active in all cases, and there's not much we can do about it. It's very good to know anyway what patterns mean the hook is not active.

Regarding "canvas", I often have issues with that package too and had to disable it in some build contexts in CI (and it's actually there for tests only so I've been considering removing it). Do you remember which version of Node you used to make it work? We should add this to the devbox file to make that clearer.